contentcuration/contentcuration/management/commands/loadconstants.py

```diff
@@ -446,6 +461,21 @@
             "allowed_formats" : [file_formats.PNG, file_formats.JPG, file_formats.JPEG],
         },
     },
+    {
+        "model": models.FormatPreset,
+        "pk": "id",
+        "fields": {
+            "id" : format_presets.HTML5_ZIP,
```

Comment:
Will we need to perform any validation on the zip files? Can we expect a standard format for these zip files in terms of what files are included or how the zip is organized?